fineract-provider/dependencies.gradle

```diff
@@ -80,7 +79,8 @@ dependencies {
 
             'com.github.spotbugs:spotbugs-annotations',
             'io.swagger.core.v3:swagger-annotations',
-            'org.webjars:webjars-locator-core'
+            'org.webjars:webjars-locator-core',
+            mysqlConnector
```

Comment:
So this, unfortunately, as I understand it, is No Go. This would mean that we include a LGPL into the distro. Or that we would have to make special handstands to remove it again when we release to distribute, which I fear will be too confusing.

What we CAN do, as far as my understanding goes, is to have a test only scope dependency. So for unit tests, it's on the classpath, but it's never "packaged" and distributed. We currently only have one Spring Integration Test, but I would still do add it, with another scope than `implementation`, but some... test only something (whatever the right name is).